internal/gitinterface/changes.go

```diff
@@ -36,17 +37,57 @@ func GetCommitFilePaths(commit *object.Commit) ([]string, error) {
 
 // GetFilePathsChangedByCommit returns the paths changed by the commit relative
 // to its parent commit. If the commit is a merge commit, i.e., it has more than
-// one parent, no changes are returned.
-//
-// Currently, this function does not verify that the tree for a merge commit
-// matches one of its parents. In a future version, this behavior may change and
-// return an error if a multi-parent commit seems invalid.
+// one parent, check if the commit is the same as at least one of its parents.
+
```

Comment:
This needs to be updated as we now return a list of changes if the tree doesn't match.

Reply:
Added. If you think anything else should be added, so it makes more sense, let me know.

Reply:
```suggestion
```